transformers/tokenization_utils.py

```diff
@@ -671,10 +671,10 @@ def convert_tokens_to_ids(self, tokens):
         ids = []
         for token in tokens:
             ids.append(self._convert_token_to_id_with_added_voc(token))
-        if len(ids) > self.max_len:
-            logger.warning("Token indices sequence length is longer than the specified maximum sequence length "
-                           "for this model ({} > {}). Running this sequence through the model will result in "
-                           "indexing errors".format(len(ids), self.max_len))
+        # if len(ids) > self.max_len:
+        #     logger.warning("Token indices sequence length is longer than the specified maximum sequence length "
+        #                    "for this model ({} > {}). Running this sequence through the model will result in "
+        #                    "indexing errors".format(len(ids), self.max_len))
```

Comment:
równie dobrze można to usunąć / dodać flagę, która wyłącza ten warning dla Twojego modelu (o ile  zrobisz własny moduł z Twoimi modyfikacjami), a dla innych modeli zostawia